research/static/research/js/dlgFileBrowseOperations.js

```diff
@@ -283,9 +393,35 @@ async function copyFolder (folderPath, newFolderpath, multiple, multipleIndex =
       }
     } else { // non api error
       if (multiple) {
-        $('.multi-select-table tr.row-' + multipleIndex + ' td.item-progress').text(result.status_info)
+        if (result.status_info.includes('already exists')) {
+          $('.multi-select-table tr.row-' + multipleIndex + ' td.item-progress').text(result.status_info + '. Do you want to overwrite?')

```

Comment:
Can we make this more readable by building the button html first and then appending it instead of building it inside the append?